bin/watch.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Script to watch for changes to actions and upload them to the recipe
+ * server.
+ */
+import 'babel-polyfill';
+
+import path from 'path';
+
+import chokidar from 'chokidar';
+import {argv} from 'yargs';
+
+import {NormandyApi} from '../lib/api';
+import {Action} from '../lib/models';
+import {loadConfig, localPath} from '../lib/utils';
+
+
+let config = loadConfig(argv.env);
+if (config === null) {
+    process.exit();
+}
+
+let api = new NormandyApi(config.api_token, config.normandy_url, config.verify);
+let watcher = chokidar.watch(localPath('actions'));
+watcher.on('change', async function(changedPath) {
+    let actionName = path.relative(localPath('actions'), changedPath).split(path.sep)[0];
+    let action = new Action(actionName);
+
+    if (action.isValid) {
```

Comment:
Should there be a else clause that prints that an action failed to load? Though, after reading the `isValid` method, I'd actually be surprised if any action that got this far wasn't valid.